app/views/foreman/unattended/finish-katello.erb

```diff
@@ -35,7 +35,7 @@ echo "updating system time"
 /usr/sbin/ntpdate -sub <%= @host.params['ntp-server'] || '0.fedora.pool.ntp.org' %>
 /usr/sbin/hwclock --systohc
 
-<% if @host.respond_to?(:realm) && @host.otp && @host.realm && @host.realm.realm_type == "FreeIPA" -%>
+<% if @host.info['parameters']['realm'] && @host.realm && @host.realm.realm_type == 'FreeIPA' -%>
```

Comment:
Nitpick:  could probably use `@host.realm.try(:realm_type) == 'FreeIPA'  `

but since it wasn't a comment on the main PR I'd not change it.